### ISSUE TITLE
More contributing informations for developpers

### ISSUE THREAD

**Issue by PierrePharel** [celluloid-player/celluloid#564]
As you can undestand with title, I write this issue for talk about informations for developper wich want contribute to project. More than bug reporting, but with writing code and more.
Sincerly PP.

**Comment by gnome-mpv** [comment]
Right now anyone is free to contribute any changes by submitting a pull request. We don't require signing a CLA or anything like that.

Do you have anything specific you want to know?

**Comment by PierrePharel** [comment]
Yes, I want an code guideline for naming conventions and, more generaly, organization of code. But above all a good project evolution view.
